plugins/rails/rails.plugin.zsh

```diff
@@ -65,7 +66,6 @@ alias rdmtc='rake db:migrate db:test:clone'
 alias rdsl='rake db:schema:load'
 alias rlc='rake log:clear'
 alias rn='rake notes'
-alias rr='rake routes'
 alias rrg='rake routes | grep'
```

Comment:
I think this should be changed too

Reply:
@Volosh1n yes, I missed that. Added a new commit.